src/zeekygen/Manager.cc

```diff
@@ -25,7 +25,7 @@ static void DbgAndWarn(const char* msg) {
         // be confusing.
         return;
 
-    reporter->Warning("%s", msg);
+    reporter->Warning("[zeekygen] %s", msg);
```

Comment:
Did you ever figure out a way to test whether this works? I only spent a few minutes trying to decipher how to make any of the parsing code trigger this function, with no success.